App.js

```diff
@@ -14,13 +14,12 @@ ProfileRoute.queries = {
 };
 
 view App {
+  prop onLogout
+
   <Relay.RootContainer
-    {...{ route: new ProfileRoute() }}
+    route={new ProfileRoute()}
```

Comment:
in your `.flint/config.js` you can turn off our routing with `routing: false`

https://github.com/reindexio/reindex-starter-kit-flint/blob/master/.flint/config.js